packages/fluent/modules/widgets/value/FluentUIMultiSelect.jsx

```diff
@@ -1,17 +1,19 @@
 import React, { useState } from "react";
 import { Dropdown } from "@fluentui/react";
 import { Utils } from "@react-awesome-query-builder/ui";
+import SearchableDropdown from "../SearchableDropdown";
+
 const { mapListValues } = Utils.ListUtils;
 
-export default ({
-  listValues,
-  value,
-  setValue,
-  allowCustomValues,
-  readonly,
-  customProps,
-  placeholder,
-}) => {
+export default (props) => {
+  const {
+    listValues,
+    value,
+    setValue,
+    readonly,
+    placeholder,
+    showSearch,
+  } = props;
   const [selectedKeys, setSelectedKeys] = useState(value ?? []);
```

Comment:
I think this line should be removed.
Thus component would not have own state and rely only on `value` prop